### ISSUE TITLE
() Quest Fervor of the Frostborn.... when you speak to Fjorlin Frostbrow, he just hops on an eagle and takes off, leaving you behind. From the quest verbiage, it sounds like he's supposed to take you with him.  I was able to fly to the quest mob, kill him, and move on, but no ride from Frostbrow. (No one seems to want me on their birbs!!) https://youtu.be/1EgnRCWXP_4?si=Sz80yEBUFA06VXYz

### ISSUE THREAD

**Issue by YggdrasilWotLK** [YggdrasilWotLK/Yggdrasil-WoW#72]
Quest Fervor of the Frostborn.... when you speak to Fjorlin Frostbrow, he just hops on an eagle and takes off, leaving you behind. From the quest verbiage, it sounds like he's supposed to take you with him.  I was able to fly to the quest mob, kill him, and move on, but no ride from Frostbrow. (No one seems to want me on their birbs!!) https://youtu.be/1EgnRCWXP_4?si=Sz80yEBUFA06VXYz

**Comment by Ye-Old-Potato** [comment]
Checked this out - whatever's up is linked to the NPC that's supposed to get you on the bird - that whole thing doesn't seem like it wants to happen at all since the NPC does nothing when you speak the required dialogue to him. Also oddly enough, there's no birds hovering about like is shown in that video either so not entirely sure what's up with it. I did try it with the completed prior quests too just to be sure, so I am at the correct point in the chain.

Note: This almost certainly has something to do with the missing static bird who isn't even spawned, the routed birds are present but the hovering one, like in the case of gryphon masters, isn't present even after completing the quest prior where you actually save said birds to be able to use em later.
